Cargo.toml

```diff
@@ -28,9 +28,18 @@ recovery = ["secp256k1-sys/recovery"]
 lowmemory = ["secp256k1-sys/lowmemory"]
 global-context = ["std"]
 
+# Give users full control of bitcoin_hashes features while stil using the secp256k1 re-exported
+# bitcoin_hashes dependency.
+bitcoin-hashes-std = ["bitcoin_hashes/std"]
+bitcoin-hashes-alloc = ["bitcoin_hashes/alloc"]
+bitcoin-hashes-std-serde = ["bitcoin_hashes/std", "bitcoin_hashes/serde-std"]
+bitcoin-hashes-alloc-serde = ["bitcoin_hashes/alloc", "bitcoin_hashes/serde"]
+bitcoin-hashes-std-schemars = ["bitcoin_hashes/std", "bitcoin_hashes/schemars"]
```

Comment:
If we're going to reexport things I think it only makes sense to reexport features that are actually used in conditions here. In this case `bitcoin-hashes-std`. However I suspect your approach may help with forward compatibility. Is it worth 2^n complexity, though?